cpp/dspace/doi.cc

```diff
@@ -0,0 +1,98 @@
+#include <iostream>
+#include <regex>
+#include <set>
+#include <unordered_map>
+#include <cstdlib>
+#include "DbConnection.h"
+#include "DbResultSet.h"
+#include "DbRow.h"
+#include "Downloader.h"
+#include "EmailSender.h"
+#include "HttpHeader.h"
+#include "JSON.h"
+#include "StlHelpers.h"
+#include "StringUtil.h"
+#include "VuFind.h"
+#include "util.h"
+
+using namespace std;
+
+const std::string DspaceServerURL("https://publikationen.uni-tuebingen.de/rest/items/");
+const std::string DOIKey("dc.relation.uri");
```

Comment:
If these are global constants, they should all be in uppercase with underscores:

```
DSPACE_SERVER_URL
DOI_KEY
PUBLICATION_TITLE_KEY
```